packages/account/src/providers/transaction-response/transaction-response.ts

```diff
@@ -260,6 +258,19 @@ export class TransactionResponse {
       transactionResult.logs = logs;
     }
 
+    if (transactionResult.isStatusFailure) {
```

Comment:
_Commenting for reference_

This logic should be moved into `this.waitForStatusChange` when implementing #1644.